content/en/docs/instrumentation/java/springboot.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Spring Boot
+linkTitle: Spring Boot
+aliases:
+  - /docs/java/getting_started
+  - /docs/java/springboot
+  - /docs/instrumentation/java/springboot
+weight: 30
+description: Spring instrumentation for OpenTelemetry Java
+javaInstrumentationVersion: 1.31.0
```

Comment:
I think you can inherit this from the _index, @chalin can you provide some guidance here?

Reply:
Yes, let me refactor a few things first before we bring in this PR.

Reply:
@svrnm see:

- #3503

PR #3503 should be merged first, and then this PR rebased and adapted to using the new key name.

Reply:
PR is merged, so what changes are needed now?

Reply:
See https://github.com/open-telemetry/opentelemetry.io/pull/3377#pullrequestreview-1727954944.